docs/ops/normalization/MVN_1.md

```diff
@@ -4,58 +4,93 @@
 
-*MVN* subtracts mean value from the input blob:
+Based on `across_channels` attribute mean value is calculated using one of formulas below:
+
+1. if `true` mean value is calculated using Layer Normalization:
+\f[
+\mu_{n} = \frac{\sum_{c}^{C}\sum_{h}^{H}\sum_{w}^{W} i_{nchw}}{C * H * W}
+\f]
+2. if `false` mean value is calculated using Instance/Contrast Normalization:
 \f[
-o_{i} = i_{i} - \frac{\sum{i_{k}}}{C * H * W}
+\mu_{nc} = \frac{\sum_{h}^{H}\sum_{w}^{W} i_{nchw}}{H * W}
 \f]
-If *normalize_variance* is set to 1, the output blob is divided by variance:
+
+where \f$i_{nchw}\f$ is an input tensor parametrized by \f$n\f$ batches, \f$c\f$ channels and \f$h,w\f$ spatial dimesnions.
+
+If `reduction_axes` attribute is provided mean value is calculated based on formula:
 \f[
-o_{i}=\frac{o_{i}}{\sum \sqrt {o_{k}^2}+\epsilon}
+\mu_{n} = ReduceMean(i_{k}, reduction_axes)
 \f]
 
+Afterwards *MVN* subtracts mean value from the input blob.
+
+If *normalize_variance* is set to `true`, the output blob is divided by variance:
+\f[
+o_{i}=\frac{o_{i}}{\sqrt {\sum {\sigma_{k}^2}+\epsilon}}
+\f]
+
+where \f$\sigma_{k}^2\f$ is the variance calculated based on mean value, \f$\epsilon\f$ is a value added to the variance for numerical stability and corresponds to `epsilon` attribute.
+
 **Attributes**
 
 * *across_channels*
 
-  * **Description**: *across_channels* is a flag that specifies whether mean values are shared across channels. For example, *across_channels* equal to `false` means that mean values are not shared across channels.
+  * **Description**: *across_channels* is a flag that specifies whether mean values are shared across channels. If `true` mean values and variance are calculated for each sample across all channels and spatial dimensions (Layer Normalization), otherwise calculation is done for each sample and for each channel across spatial dimensions (Instance/Contrast Normalization).
   * **Range of values**:
     * `false` - do not share mean values across channels
     * `true` - share mean values across channels
   * **Type**: `boolean`
-  * **Default value**: `false`
+  * **Default value**: None
+  * **Required**: *no*
+
+* *reduction_axes*
+
+  * **Description**: 1D tensor of unique elements and type *T_IND* which specifies indices of dimensions in `data` that define normalization slices. Negative value means counting dimensions from the back.
+  * **Range of values**: allowed range of axes is `[-r; r-1]` where `r = rank(data)`, the order cannot be sorted
+  * **Type**: `int`
+  * **Default value**: None
   * **Required**: *no*
 
 * *normalize_variance*
 
   * **Description**: *normalize_variance* is a flag that specifies whether to perform variance normalization.
   * **Range of values**:
-    * `false` -- do not normalize variance
-    * `true` -- normalize variance
+    * `false` - do not normalize variance
+    * `true` - normalize variance
   * **Type**: `boolean`
-  * **Default value**: `false`
+  * **Default value**: None
```

Comment:
if this is not mandatory attribute, the default value must exist

Reply:
Fixed.